runtime/runtime_controller.h

```diff
@@ -191,6 +203,9 @@ class RuntimeController : public PlatformConfigurationClient {
   ///
   /// @param[in]  view_id  The ID of the view.
   ///
+  /// @return     If the remove view operation was forwarded to the running
+  ///             isolate.
```

Comment:
Copying this conversation: https://github.com/flutter/engine/pull/51523#discussion_r1538203021

> @[dkwingsmt](https://github.com/dkwingsmt) (https://github.com/flutter/engine/pull/51523#discussion_r1538203021)
> This explanation is very hard to understand to me. Can it simply be "Whether the removal is successful."?
> 
> @[loic-sharma](https://github.com/loic-sharma) (https://github.com/flutter/engine/pull/51523#discussion_r1538335582)
> This is the language used by other similar methods on this class:
> 
> [engine/runtime/runtime_controller.h](https://github.com/flutter/engine/blob/b2001fa6d7738752294aea4c2fb87486cf84c174/runtime/runtime_controller.h#L204-L206)
>
> > Can it simply be "Whether the removal is successful."?
> 
> There's a subtle difference: if the Dart isolate isn't launched, this succeeds and returns false. If we want to change this behavior, I would say that's better kept to a separate pull request :)